src/main/java/net/sf/jabref/logic/bibtexkeypattern/BibtexKeyPatternUtil.java

```diff
@@ -692,7 +665,7 @@ else if (val.matches("edtr\\d+")) {
                 String ss = entry.getFieldOrAlias(FieldName.YEAR).orElse("");
                 if (ss.isEmpty()) {
                     return ss;
-                } else if (ss.startsWith("in") || ss.startsWith("sub")) {
+                } else if (ss.startsWith("in") || ss.startsWith("sub")) { // TODO: Year starting with in or sub? Why?
```

Comment:
No idea why sub or in either. Any idea why the var is called "ss"? I would change it to a more expressive variable name